xml/installation-installation-ses_integration.xml

```diff
@@ -220,51 +166,65 @@
     nova:
         rbd_store_pool: nova
 
-    glance:
-        rbd_store_pool: glance
+    glance:														
+        rbd_store_pool: glance												
         rbd_store_pool_user: glance
         keyring_file_name: ceph.client.glance.keyring</screen>
-  </example>
-  <para>
-   The path to this directory must be specified in the
-   <filename>settings.yml</filename> file, as in the example below. After
-   making the changes, follow the steps to complete the configuration.
-  </para>
-<screen>settings.yml
-...
-ses_config_path: /var/lib/ardana/ses/
-ses_config_file: ses_config.yml
 
-# The unique uuid for use with virsh for cinder and nova
- ses_secret_id: <replaceable>SES_SECRET_ID</replaceable></screen>
-  <procedure>
-   <step>
-    <para>
-     After modifying these files, commit your configuration to the local git
-     repo. For more information, see <xref linkend="using-git"/>.
-    </para>
-<screen>&prompt.ardana;cd ~/openstack/ardana/ansible
+	<para>
+	 Example contents of the directory specified in <filename>settings.yml</filename> file:
+	</para>
+
+<screen>ardana > ~/openstack/my_cloud/config/ses> ls -al
+ceph.client.cinder-backup.keyring											
+ceph.client.cinder.keyring												
+ceph.client.glance.keyring					
+ses_config.yml</screen>
+
+<para>
+ Modify the <literal>glance_default_store</literal> option in <filename>~/openstack/my_cloud/definition/data/control_plane.yml</filename>:
+</para>
+
+<screen>.
```

Comment:
I didnt notice it before, but this screen section looks a little weird... commenting out one line just above an identical non-commented-out duplicate of that line? the dots above the sections I assume are meant to represent a scroll of unrelated content, but I'm not sure we do that in other screen sections.

Reply:
I missed the colon in the new `glance-api:` line... so maybe this is ok as-is